test/distributed/test_jit_c10d.py

```diff
@@ -144,6 +144,20 @@ def forward(self, input: torch.Tensor):
         self.checkModule(TestModule(pg_nccl), (torch.rand((2, 3)),))
 
 
+class StoreTest(JitTestCase):
+    def setUp(self):
+        super(StoreTest, self).setUp()
+        self.file = tempfile.NamedTemporaryFile(delete=False)
+        self.filestore = torch.classes.dist_c10d.FileStore(self.file.name, 1)
```

Comment:
Shouldn't we test that calling this within JIT works appropriately? Or is there some magic in `JitTestCase` that is already doing this?

Reply:
nothing magic, the internal use case right now only testing if python side works when creating the process group, but it's a better idea that we also test the JIT side store creation works fine, updating.